core/src/main/scala/stryker4s/mutants/findmutants/MutantMatcher.scala

```diff
@@ -2,54 +2,54 @@ package stryker4s.mutants.findmutants
 
 import stryker4s.extensions.ImplicitMutationConversion.mutationToTree
 import stryker4s.extensions.mutationtypes._
-import stryker4s.model.FoundMutant
+import stryker4s.model.Mutant
 
 import scala.meta.Tree
 
-class MutantMatcher {
+class MutantMatcher extends MutantCreator {
```

Comment:
Well as I was not sure if I wanted to include it into the `MutantMatcher` or putt it into a trait. I just putt it into a trait because of separation of concerns. But either works for me.

An `Object` is not an option though since there will only be one then and the mutant id's will be all over the place in the tests which will break a lot and make it all shakey.